lib/logthrdestdrv.c

```diff
@@ -32,13 +32,13 @@
 void
 log_threaded_dest_driver_set_max_retries(LogDriver *s, gint max_retries)
 {
-  LogThrDestDriver *self = (LogThrDestDriver *)s;
+  LogThreadedDestDriver *self = (LogThreadedDestDriver *)s;
```

Comment:
The "Threaded" name in `LogThreadedDestDriver` is a little unfortunate. I had a preconception due to this for quite some time, thinking LogThreadedDestDriver is a destination driver that works with multiple threads, hence the name. Then it turned out it is still single thread, just the thread is not the main loop. Here the name carries some implementation detail, instead of the purpose of the concept. As it is natural expectation that a worker is a thread, we could just drop Threaded, instead of making it more explicit. We could call DestinationDriver (LogDestDriver is already taken)?